### ISSUE TITLE
Found what is crashing UI

### ISSUE THREAD

**Issue by erickiwa** [wisp-forest/owo-lib#186]
today i was tryng to use interact mod but i aways got the same [Error](https://imgur.com/a/bewtZwW)

it was simples, optfabric(or optfine) overlaps owolib. I have seen some issues about it and I also think that other mods that change the interface indirectly cause conflict.

(i'm using an outdated fabric api because 
the latest one is not compatible with optfabric)

**Comment by BasiqueEvangelist** [comment]
I'm confused. Are the 3 images in the imgur album with the same modlist, or does one of them have Optifine disabled?

**Comment by erickiwa** [comment]
> I'm confused. Are the 3 images in the imgur album with the same modlist, or does one of them have Optifine disabled?

Yes, imagine a sequence from bottom to top. The solution for now is just not to use optfine. Other than that, very few mods will cause this problem, if you are using fabric there are even better solutions than optfine itself.
